sympy/polys/ring_series.py

```diff
@@ -1748,3 +1787,118 @@ def rs_compose_add(p1, p2):
     if dp:
         q = q*x**dp
     return q
+
+
+_convert_func = {
+        'sin': 'rs_sin',
+        'cos': 'rs_cos',
+        'exp': 'rs_exp',
+        'tan': 'rs_tan'
+        }
+
+def rs_min_pow(expr, series_rs, a):
+    series = 0
+    n = 2
+    while series == 0:
+        series = _rs_series(expr, series_rs, a, n)
+        n *= 2
+    R = series.ring
+    a = R(a)
+    i = R.gens.index(a)
+    return min(series, key=lambda t: t[i])[i]
+
+
+def _rs_series(expr, series_rs, a, prec):
+    # XXX Use _parallel_dict_from_expr instead of sring.
```

Comment:
I would change XXX to TODO, and perhaps explain more about the background why the TODO item is needed, so that other people can pick it up if needed.